thriftpy2/http.py

```diff
@@ -159,7 +177,7 @@ class THttpClient(object):
     """Http implementation of TTransport base.
     """
 
-    def __init__(self, uri, timeout=None, ssl_context_factory=None):
+    def __init__(self, uri, timeout=None, ssl_context_factory=None, http_header_factory=THttpHeaderFactory()):
```

Comment:
Default value in a keyword arguments is actually static (global) variable, which will leads to misleading usage. I think it's better to use None as the default value, and check if it's None and initialize it in function body.

Reply:
Fixed in latest update